WordPress/src/test/java/org/wordpress/android/sharedlogin/resolver/SharedLoginResolverTest.kt

```diff
@@ -106,10 +116,23 @@ class SharedLoginResolverTest {
     }
 
     @Test
-    fun `Should dispatch UpdateTokenPayload if access token is NOT empty`() {
+    fun `Should dispatch UpdateTokenPayload if access token is NOT, flags and saved posts are migrated`() {
         featureEnabled()
+        onSuccessFlagsCaptor = argumentCaptor()
+        onSuccessRaaderPostsCaptor = argumentCaptor()
```

Comment:
I think there was a typo in this variable name (`onSuccessRaaderPostsCaptor` -> `onSuccessReaderPostsCaptor`).